### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by mojoLyon**

### CODE REVIEW

**Comment by cordoval** [comment]
would be nice to explain some background and use case, i want to use this bundle but the goal is not very clear in terms of a problem that solves this well, even with the driver and all

**Comment by omansour** [comment]
@cordoval the bundle allow you to use the datastax cassandra component in sf2 and basically request cassandra database. Feel free to enhance the README via a pull request 

@mojoLyon :+1:

File: README.md [hunk 1]
```diff
@@ -6,28 +6,149 @@ The CassandraBundle provide a Cassandra client as a Symfony service.
...
+Then install the bundle :
+
+```shell
+$ composer update m6web/cassandra-bundle
+```
 
 ## Usage
 
-coming soon
+Add the `m6web_cassandra` section in your configuration file. Here the minimal configuration required.
```
Comment (mguillermin) [inline README.md]: s/Here the minimal/Here is the minimal

File: README.md [hunk 2]
```diff
@@ -6,28 +6,149 @@ The CassandraBundle provide a Cassandra client as a Symfony service.
...
+// do something while cassandra query running
+
+foreach($result->get() as $row) {
+    // do something with row
+}
+```
+
+## DataCollector
+
+Datacollector is available when the symfony profiler is enabled. The colector allow you to see the following Cassandra data :
```
Comment (mguillermin) [inline README.md]: s/allow/allows

File: README.md [hunk 3]
```diff
@@ -6,28 +6,149 @@ The CassandraBundle provide a Cassandra client as a Symfony service.
...
+
+- keyspace
+- command name
+- command arguments
+- execution time
+- execution options override (consistency, serial consistency, page size and timeout)
+
+**NOTE :** The time reported in data collector may not be the real execution time in case you use the async calls : `executeAsync` and `prepareAsync`
```
Comment (mguillermin) [inline README.md]: s/in data collector/in the data collector

**Comment by mguillermin** [comment]
Apart from my minor comments, :+1:

File: README.md (hunk 2 above)
Comment (AntoineReneleau) [inline README.md]: s/colector/collector

**Pull request merged**